portfolio/src/main/webapp/script.js

```diff
@@ -12,20 +12,103 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['I am 20 years old!', 'My favorite color is pink!', 
-      'I am in Phi Sigma Sigma!', 'I used to work at John Deere!',
-      'I am outdoorsy!', 'My favorite race is the 5k!',
-      'I have done other Google programs before!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// /**
+//  * Adds a random greeting to the page.
+//  */
+// function addRandomGreeting() {
+//   const greetings =
+//       ['I am 20 years old!', 'My favorite color is pink!',
+//       'I am in Phi Sigma Sigma!', 'I used to work at John Deere!',
+//       'I am outdoorsy!', 'My favorite race is the 5k!',
+//       'I have done other Google programs before!'];
+
+//   // Pick a random greeting.
+//   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+
+//   // Add it to the page.
+//   const greetingContainer = document.getElementById('greeting-container');
+//   greetingContainer.innerText = greeting;
+// }
+
+// async function getText() {
+//   const response = await fetch('/data');
+//   const txt = await response.text();
+//   document.getElementById('txt-container').innerText = txt;
+// }
+
+// function getMessages() {
+//         console.log("Made it here 1");
+//   fetch('/data').then(response => response.json()).then((messages) => {
+//     // messages is an object, not a string, so we have to
+//     // reference its fields to create HTML content
+
+//     console.log("Made it here 2");
+
+//     const messagesElement = document.getElementById('messages-container');
+//     messagesElement.innerHTML = '';
+//     messagesElement.appendChild(
+//         createListElement('1: ' + messages[0]));
+//     messagesElement.appendChild(
+//         createListElement('2: ' + messages[1]));
+//     messagesElement.appendChild(
+//         createListElement('3: ' + messages[2]));
+//   });
+// }
+
+// function createListElement(text) {
+//   const liElement = document.createElement('li');
+//   liElement.innerText = text;
+//   return liElement;
+// }
+
+/** Fetches comments from the server and adds them to the DOM. */
+function loadComments() {
+  const value = document.getElementById("num-comments").value;
+  fetch(`/list-comments?num-comments=${value}`)
+    .then((response) => response.json())
+    .then((comments) => {
```

Comment:
You may want to use 'async' and 'await' so that you can eliminate using 'then' and the code is a little easier to read.